README.md

```diff
@@ -8,12 +8,42 @@ See [the guide](https://mycorrhiza.lesarbr.es/hypha/guide/deployment) on the wik
 
 ## Installing
 
+### Pacman
+
 If you use a linux distro with pacman package manager (Arch, Manjaro, Garuda, etc) you can install it using PKGBUILD:
 ```sh
 $ wget https://raw.githubusercontent.com/bouncepaw/mycorrhiza/master/PKGBUILD
 $ makepkg --install
 ```
 
+### Docker
+
+Use Dockerfile:
```

Comment:
"You can run Mycorrhiza Wiki in Docker using Dockerfile provided by this repository. Clone the repo and build the container:"